doc/content/guides/interop-repository/_index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Interoperability Repository"
+description: ""
+weight: 40
+---
+
+## Interoperability Repository
+
+In order to enable interoperability in the stack:
+1. Create an interoperability configuration repository (e.g. a directory) with `config.yaml` at the root
```

Comment:
Isn't it a bit confusing that we use `.yaml` instead of `.yml` like everywhere else?

Reply:
It's too late to change that.